QualifiedTeachersApi/src/QualifiedTeachersApi/Infrastructure/OpenApi/HandleOptionJsonSchemaGenerator.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Namotion.Reflection;
+using NJsonSchema;
+using NJsonSchema.Generation;
+using NSwag.Generation;
+using Optional;
+
+namespace QualifiedTeachersApi.Infrastructure.OpenApi;
+
+public class HandleOptionJsonSchemaGenerator : OpenApiSchemaGenerator
+{
+    public HandleOptionJsonSchemaGenerator(OpenApiDocumentGeneratorSettings settings)
+        : base(settings)
+    {
+    }
+
+    public override TSchemaType GenerateWithReferenceAndNullability<TSchemaType>(
+        ContextualType contextualType,
+        bool isNullable,
+        JsonSchemaResolver schemaResolver,
+        Action<TSchemaType, JsonSchema>? transformation = null)
+    {
+        if (contextualType.Type.IsGenericType && contextualType.Type.GetGenericTypeDefinition() == typeof(Option<>))
```

Comment:
This makes `Option<T>` properties appear the same as `T` in the generated specs.